src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs

```diff
@@ -316,14 +317,18 @@ private async Task LoadKeyValuesRegisteredForRefresh(IDictionary<string, Configu
             {
                 string watchedKey = changeWatcher.Key;
                 string watchedLabel = changeWatcher.Label;
+                KeyLabel watchedKeyLabel = new KeyLabel(watchedKey, watchedLabel);

```

Comment:
>KeyLabel [](start = 16, length = 8)

It has properties `Key` and `Label`, but I'm thinking the type itself is more of `KeyValueIdentifier`.